lib/ecto_mnesia/planner.ex

```diff
@@ -6,6 +6,8 @@ defmodule EctoMnesia.Planner do
   alias :mnesia, as: Mnesia
   alias EctoMnesia.{Record, Table}
   alias EctoMnesia.Record.{Context, Ordering, Update}
+  alias Ecto.Adapters.SQL
+  alias Ecto.UUID
```

Comment:
I'd prefer to don't alias them on purpose: when we use eternal private API we want to be 100% explicit in the place where we use it.